src/race/Validator.js

```diff
@@ -0,0 +1,39 @@
+import { ERROR_MESSAGE } from "../utils/ErrorMessage.js";
+
+export function checkValidNameLength(name, maxLength) {
+    if (name.length > maxLength) {
+        throw new Error(ERROR_MESSAGE.NAME_LENGTH_EXCEEDED(maxLength, name));
+    }
+}
+
+export function checkNotBlank(str) {
+    if (!str) {
+        throw new Error(ERROR_MESSAGE.EMPTY_INPUT_NOT_ALLOW);
+    }
+}
+
+export function checkOnlyEnglishAndNumberCharacters(str) {
```

Comment:
저는 함수명에 `end`나 `or`이 들어가야 하는 경우에 함수 분리를 고민하는 편이에요!

`end`나 `or`이  함수가 두 가지 역할을 담당하고 있다는 것을 그대로 드러내는 거 같아서 그런거 같습니다.

지금의 방법도 좋지만 아래와 같이 접근 해볼 수도 있을 거 같아요!

```suggestion
export function checkIsValidCharacters(input) {
  checkOnlyEnglishCharacters(input);
  checkOnlyNumberCharacters(input);
}
```

유효성 검사를 분리해서 함수의 재활용을 높이고 구체적인 유효성 검사에 대한 문맥은 함수 내부로 숨겨서 유효성을 검사한다는 부분만 알 수 있게 추상화 해봤습니다!